configs/katello_36.yaml

```diff
@@ -0,0 +1,181 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.5.0:
+    :redmine_release_id: 250
+  :3.5.1:
+    :redmine_release_id: 329
+:releases:
+  :3.6.0:
+    :redmine_release_id: 284
+:code_name: Imperial IPA
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.6
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :katello-selinux:
+    :branch: 3.0.3
+    :repo: https://github.com/Katello/katello-selinux.git
+  :katello-installer:
+    :branch: KATELLO-3.6
+    :repo: https://github.com/Katello/katello-installer.git
+    :version_branch: true
+  :hammer-cli-katello:
+    :branch: 0.11.4
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :foreman-packaging:
+    :branch: rpm/1.17
+    :repo: https://github.com/theforeman/foreman-packaging.git
+    :version_branch: true
```

Comment:
@akofink Since we don't control this repository from the Katello perspective, I think we'd want to drop this right?

Reply:
By 'this' I mean the flag here

Reply:
Oh yeah, you're right